questionary/prompts/text.py

```diff
@@ -52,14 +59,21 @@ def text(
 
     validator = build_validator(validate)
 
+    if instruction is None and multiline:
+        instruction = "(Finish with 'Alt+Enter' or 'Esc, Enter')"
```

Comment:
How am I supposed to assert that a tty displays this message properly?

Reply:
Good question - I should have looked more closely at the lines being highlighted! Don't think a test is required in this case then.